x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -232,12 +240,6 @@ export const ConfigurePipeline: React.FC = () => {
                     disabled={inputsDisabled}
                     fullWidth
                     prepend="ml-inference-"
-                    placeholder={i18n.translate(
```

Comment:
there is no need to remove the placeholder here - if someone deletes the default we prepopulate, the placeholder still makes sense IMO

<img width="1150" alt="Screenshot 2023-08-03 at 11 52 30" src="https://github.com/elastic/kibana/assets/16032709/b3a26ffa-02b5-4200-b504-ceffa4fb3afd">
<img width="1151" alt="Screenshot 2023-08-03 at 11 53 37" src="https://github.com/elastic/kibana/assets/16032709/618bd1e2-1851-4ec8-8bc2-a6b8972a384a">

Reply:
Good catch. I will change this part accordingly.